core/src/main/java/org/ehcache/core/statistics/CacheStatistics.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.core.statistics;
+
+import java.util.Map;
+
+/**
+ * All statistics relative to a cache and its underlying tiers.
+ *
+ * @author Henri Tremblay
```

Comment:
Please configure your IDE to not generate these, the source control systems does a much better job